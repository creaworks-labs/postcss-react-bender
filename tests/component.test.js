var { processFileAndMatchSnapshot } = require("./utils");

it("process component based styles", () => {
  return processFileAndMatchSnapshot("./tests/component.basic.scss");
});

it("process nested component based styles", () => {
  return processFileAndMatchSnapshot("./tests/component.nested.scss");
});
