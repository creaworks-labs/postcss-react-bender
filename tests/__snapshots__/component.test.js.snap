// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`process component based styles 1`] = `
Object {
  "developer.project.view": Object {
    "backgroundColor": "red",
    "height": 50,
    "width": 100,
  },
}
`;

exports[`process nested component based styles 1`] = `
Object {
  "developer.project.screen": Object {
    ".square": Object {
      "alignItems": "center",
      "backgroundColor": "yellow",
      "height": 55,
      "justifyContent": "center",
      "marginTop": "10pt",
      "width": 25,
    },
    "developer.project.view": Object {
      "backgroundColor": "red",
      "height": 50,
      "width": 100,
    },
    "fontSize": 25,
  },
}
`;
